clients/go/pkg/worker/jobPoller.go

```diff
@@ -16,117 +16,131 @@
 package worker
 
 import (
-	"context"
```

Comment:
Not sure, how it got the wide indent earlier. As far as I can tell my Goland respects the editorconfig and formats the code correctly. 

This makes the review a bit difficult.

Reply:
🤔 go fmt isn't really run or checked explicitly anywhere in our CI. And the editorconfig exists in the root of the repo, which may not be visible to IDEs that open the `clients/go` folder as the root folder.

💭 Strangely though our `.editorconfig` does not specify `indent_size` for go files.

💭 I'd prefer automated CI for style changes, but it seems we don't have it so 🤷